### PR TITLE
Settings dashboard

### PR DESCRIPTION
**Pull request by jelbin313**
Made the front end changes discussed in class. Also created an about page, but there is nothing more than filler text on it right now

### CODE REVIEW

**Comment by Qtipshooter** [comment]
I don't see in way to get to the about page other than to directly type in the route. Were we going to have that on the settings page?

**Comment by jelbin313** [comment]
> I don't see in way to get to the about page other than to directly type in the route. Were we going to have that on the settings page?

@Qtipshooter We didn't agree on how we would get to the about page in class, so I figured I should ask in class before I just stuck a button somewhere.

File: templates/banner.html [hunk 1]
```diff
@@ -7,9 +7,12 @@
         <a class="navbar-brand homepage-navbar-title" href="{{url_for('dashboard')}}" style="margin-left: 15%;"><i class="bi bi-journal"></i> TASKBOOK</a>
       {% endif %}
       <form class="d-flex d-none d-sm-none d-md-block" style="margin-right: 15%;">
+        <!--button for settings, only appears when logged in-->
         <!-- we had url_for('login') before but we need that auth.login because login route is inside the auth view (auth.py) -->
         <!-- if user is logged in then show logout button else show login button -->
         {% if(session['user_email']) %}
+          <!--button for settings, only appears when logged in-->
+          <a class="btn btn-yellow" href="{{url_for('settings')}}">Settings</a>
```
[the remark this answers is not in the document]
Comment (jelbin313) [reply templates/banner.html:15]: I can do that

[the remark this answers is not in the document]
Comment (jelbin313) [reply templates/banner.html:15]: @Qtipshooter Done, let me know if I should add more space

**Comment by Qtipshooter** [comment]
Hmm..  Maybe down on the footer for now then since we haven't decided on a place.  I can't picture it anywhere else, but I just don't want to have a page we can only get to by directly typing it in.  The button spacing looks a lot better, thank you

**Review by Qtipshooter — APPROVED** [review]
I still want to add a spot at the bottom for the about page, but this can still be put in and it will be alright.  Just don't let us forget about the about page missing link

**Pull request merged**